bvm/ballerina-runtime/src/main/resources/MessagesBundle.properties

```diff
@@ -263,3 +263,11 @@ regexp.invalid.unicode.general.category.value = invalid Unicode general category
 regexp.invalid.unicode.property.value = invalid Unicode property value ''{0}''
 regexp.empty.character.class.disallowed = empty character class disallowed
 regexp.invalid.hex.digit = invalid hexadecimal digit
+config.env.vars.ambiguity = configurable value for variable ''{0}'' clashes with multiple environment variables {1}
+config.env.variable.ambiguity = configurable value for variable ''{0}'' clashes with variable ''{1}''. Please \
+  provide the env variable as ''[{2}]''
```

Comment:
Shall we make this error message as single sentence?

Reply:
IMO, having two sentences is more clear and understandable, specifically in configurable cases.
The sentences are separated because one explains the error and the other suggests a solution. The error message becomes lengthy and unclear when I combine both. 

Do you have any suggestions?

Reply:
We can keep as it is.